kong.conf.default

```diff
@@ -185,6 +185,27 @@
                                   # `text/html`, `application/json`, and
                                   # `application/xml`.
 
+#client_max_body_size = 0        # Defines the maximum request body size allowed
```

Comment:
Ah right, we have this hard-coded right now... We should probably introduce that change in 0.11 indeed. `0` is a dangerous default here, and contrary to our "least trust" policy introduced with 0.11 for saner/safer defaults.